src/app/content/highlights/components/HighlightStyles.tsx

```diff
@@ -129,28 +123,8 @@ export const ImageWrapper = styled.div`
   width: ${(desktopPopupWidth - popupBodyPadding) / 2}rem;
 `;
 
-// tslint:disable-next-line: variable-name
-export const StickyNoteBullet = styled.div`
```

Comment:
Is there a reason why it was left out?

Reply:
Why were the bullets removed? @TomWoodward

Reply:
i think the "bullet" you removed is actually the point on the sticky note, which we want to keep